addon/index.js

```diff
@@ -103,12 +145,20 @@ let RouterScrollMixin = Mixin.create({
     }
 
     const delayScrollTop = get(this, 'service.delayScrollTop');
+    const afterPaint = get(this, 'service.afterPaint');
+    const afterIdle = get(this, 'service.afterIdle');
```

Comment:
Both of these aren't strictly necessary for this PR.  Likely users won't want either (b/c it leaves artifacts for most route transition).  However, I added and documented just to test/make sure this PR does what is says it does.